examples/firewall/README.md

```diff
@@ -0,0 +1,30 @@
+Firewall
+==
+The Firewall NF is an NF which drops/forwards packets based on LPM rules.
+
+Compilation and Execution
+--
+```
+cd examples
+make
+cd firewall
+./go.sh INSTANCE_ID -d SERVICE_ID -f RULES_FILE
+
+OR
+
+./go.sh INSTANCE_ID -d SERVICE_ID -f RULES_FILE -b 
```

Comment:
no need to provide this example, we only provide ways to start the NF, this just has the `-b` flag